### DISCUSSION TITLE
Windows Terminal v1.17.11461.0

### DISCUSSION THREAD

**Discussion by DHowett** [microsoft/terminal#15473] [Announcements]
Enjoy this quick bug fix release!

* The "Developer Command Prompt" profiles are no longer totally busted (#15439)
* Terminal no longer crashes when you have `focusFollowMouse` mode enabled (#15420)

---

### Asset Hashes

- Microsoft.WindowsTerminal_1.17.11461.0_8wekyb3d8bbwe.msixbundle
   - SHA256 `E3A5A581F2A404D9D97F6826890BE9FD7BFC827498035D6C45E1C12F9D2D4E8F`
- Microsoft.WindowsTerminal_1.17.11461.0_8wekyb3d8bbwe.msixbundle_Windows10_PreinstallKit.zip
   - SHA256 `2BB87BC4EA358F58D1F1BF69486B133DB09EC509BF07A7BD740F9ABBE2F25CB5`
- Microsoft.WindowsTerminal_1.17.11461.0_arm64.zip
   - SHA256 `2CB999CC7AF12245E08D57D5472571C126A411C38E20F8D66E2110650FD0D757`
- Microsoft.WindowsTerminal_1.17.11461.0_x64.zip
   - SHA256 `F2B1539649D17752888D7944F97D6372F8D48EB1CEB024501DF8D8E9D3352F25`
- Microsoft.WindowsTerminal_1.17.11461.0_x86.zip
   - SHA256 `6A052D0501680EEADF7971B72734AC0084ECBD65B0A420E3C49014BE1860E070`

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/terminal/releases/tag/v1.17.11461.0'>Windows Terminal v1.17.11461.0</a>.</em>

**Comment by evilgordita** [comment]
Are there any potential reasons why Terminal wouldn't update from the store? I'm still on 1.16.10261.0 on both of my Windows 10 machines (Pro and Home respectively). I just now tried uninstalling and installing fresh from the store and it's still giving me 1.16.10261.0. I don't have the preview installed. Is delayed rollout a thing or is something broken with the release in the store?

I can download the GitHub release but I'm not too fussed over it right now.

**Reply by DHowett** [reply]
Thanks for asking! We're doing a delayed rollout for all Stable channel users, just to make sure we don't have a terrible bug that's gonna break a few million machines :smile:

**Reply by tusharsnx** [reply]
Oh I just checked that my terminal is also on v1.16, I thought we are on v1.15 as the store page is still showing v1.15 in 'What's new in this version' section. I think that needs to be updated or completely removed to not cause any confusion 🤦‍♂️

**Reply by skst** [reply]
@tusharsnn, if you scroll down farther on the Store page, it displays the _Installed version_ as **1.16.10262.0**. I agree the _What's New_ section could use a little love though. :)

@DHowett, if we download `Microsoft.WindowsTerminal_1.17.11461.0_8wekyb3d8bbwe.msixbundle` and install it, will it replace the Store version and still allow us to install the next upgrade from the Store or will it install side-by-side with the Store version?

**Reply by DHowett** [reply]
> @DHowett, if we download `Microsoft.WindowsTerminal_1.17.11461.0_8wekyb3d8bbwe.msixbundle` and install it, will it replace the Store version and still allow us to install the next upgrade from the Store or will it install side-by-side with the Store version?

@skst Yep, this will work as expected! It will upgrade the store-installed version, and the store will upgrade over it next time there's an even newer release.

**Comment by DHowett** [comment]
This version is now officially rolling out to 100% of users!

**Comment by melanchall** [comment]
Hello @DHowett,

I've downloaded the [Microsoft.WindowsTerminal_1.17.11461.0_8wekyb3d8bbwe.msixbundle](https://github.com/microsoft/terminal/releases/download/v1.17.11461.0/Microsoft.WindowsTerminal_1.17.11461.0_8wekyb3d8bbwe.msixbundle) file but I can't install the Terminal on my Windows 10 (22H2 / 19045.2965). The error is

```
This app package is not supported for installation by App Installer because it uses certain restricted capabilities.
```

Can you please help with the installations? Is Win 10 supported?

Thank you,
Max

**Reply by j0le** [reply]
I've the same Windows version and get the same error message, but in German:
```
Für dieses App-Paket wird die Installation per App-Installer nicht unterstützt, da bestimmte eingeschränkte Funktionen verwendet werden.
```
On my machine the Microsoft Store is disabled by company policy or something like that, but I could install the msixbundle files in the past. My current Windows Terminal version is 1.16.10231.0.

**Reply by j0le** [reply]
Running Powershell as Admin gave me this error:
```
Add-AppxPackage -Path .\Microsoft.WindowsTerminal_1.17.11461.0_8wekyb3d8bbwe.msixbundle
```
```
Add-AppxPackage : Deployment failed with HRESULT: 0x80073CF3, Package failed updates, dependency or conflict
validation.
Windows cannot install package Microsoft.WindowsTerminal_1.17.11461.0_x64__8wekyb3d8bbwe because this package depends
on a framework that could not be found. Provide the framework "Microsoft.UI.Xaml.2.7" published by "CN=Microsoft
Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US", with neutral or x64 processor architecture and
minimum version 7.2208.15002.0, along with this package to install. The frameworks with name "Microsoft.UI.Xaml.2.7"
currently installed are: {}
NOTE: For additional information, look for [ActivityId] 6e19c2f6-9ae9-0005-ac53-1b6ee99ad901 in the Event Log or use
the command line Get-AppPackageLog -ActivityID 6e19c2f6-9ae9-0005-ac53-1b6ee99ad901
At line:1 char:1
+ Add-AppxPackage -Path .\Microsoft.WindowsTerminal_1.17.11461.0_8wekyb ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : WriteError: (C:\Users\JanH\D...bbwe.msixbundle:String) [Add-AppxPackage], IOException
    + FullyQualifiedErrorId : DeploymentError,Microsoft.Windows.Appx.PackageManager.Commands.AddAppxPackageCommand
```

Which pointed me to the **solution**: https://answers.microsoft.com/en-us/windows/forum/all/what-is-microsoftuixaml27-and-why-dont-i-have-it/9e5753be-3b5f-4975-ac00-a28344c710a6

**Reply by DHowett** [reply]
If you download the "Preinstallation Kit" from our Releases page, it comes with all of the dependencies you will need. :)

**Reply by DHowett** [reply]
@melanchall sorry about that error. You should be able to install it using `Add-AppxPackage` as noted above.

**Reply by j0le** [reply]
> If you download the "Preinstallation Kit" from our Releases page, it comes with all of the dependencies you will need. :)

Thank you, @DHowett! That's a good tip. Next time I will know and try that.

**Reply by melanchall** [reply]
Thanks @DHowett, that works! Just a small question: so you won't provide a simple msixbundle file that can be installed by double click on Win 10 anymore?

**Reply by DHowett** [reply]
We tested this out before shipping it on Windows 10, and it worked fine for simple double-click install! We believe that it requires the newest version of the "Desktop App Installer," which is the package that provides the UI you see when double-clicking msixbundle files. (It also contains winget, which is perhaps more interesting.)

**Comment by jabaya91** [comment]
Howdy ya'll,

I was having some trouble with my windows terminal so I ended up uninstalling it(used REVO) and reinstalling it again to this version. I am running into two issues. One of which is `wt` command not opening up the terminal itself. This may have something to do with how it is named. Shows it to be "WT.exe" on my work computer; at home the application is named just "wt" Having trouble renaming it and im sure it just needs some permissions to be tweaked, but if you have any insight great. Secondly, my main issue, i cannot open admin instances from within the console. The UAC comes up as unknown program. I click allow and nothing happens. I can open up the whatever console in admin itself but not from within the terminal. Thank you in advance!

**Reply by zadjii-msft** [reply]
Sorry, just seeing this now. 

Please for the love of god, don't try and fuck with the permissions for the app package. Others have tried to before, and oftentimes it ends up with a machine in a permenantly busted state. The app package permissions are _very_ sensitive. 

> The UAC comes up as unknown program

That's #12357, which is the internal bug MSFT:37540496. I think that was fixed and being serviced, but it's unclear which versions of Windows exactly have that fix.

> i cannot open admin instances from within the console ... I click allow and nothing happens

Sounds a bit like #15576

**Comment by m0rph0** [comment]
Hi Dustin, can you have a look at Pull request 108350 for Windows Terminal 1.17, it's been waiting for actions since May 29th.. Thanks!

**Reply by zadjii-msft** [reply]
Sorry... which PR? We're only in the 15xxx range of number of total issues 😅

**Reply by m0rph0** [reply]
https://github.com/microsoft/winget-pkgs/pull/108350

**Reply by zadjii-msft** [reply]
That looks like it's still blocked on the upstream https://github.com/microsoft/winget-cli/issues/163.
